PancakeSwap-Bots/Lottery-Admin-Operations/src/findings.ts

```diff
@@ -0,0 +1,32 @@
+import { Finding, FindingType, FindingSeverity } from "forta-agent";
+import { ALERTS } from "./abi";
+
+export const createEventFinding = (_name: string, _description: string, _metadata: {}) => {
+  return Finding.fromObject({
+    name: `Event: ${_name === "NewRandomGenerator" ? "NewRandomGenerator" : _name}`,
+    description: `PancakeSwapLottery: ${_description}`,
```

Comment:
```suggestion
    description: `PancakeSwapLottery event (${_description}) emitted`,
```
continuation of the name suggestion ^